packages/shared-state-wifi_links_info/files/etc/uci-defaults/shared-state_wifi_links_info

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+unique_append()
+{
+	grep -qF "$1" "$2" || echo "$1" >> "$2"
+}
+
+unique_append \
+	'*/5 * * * * ((sleep $(($(random-numgen) % 120)); shared-state sync wifi_links_info &> /dev/null)&)'\
```

Comment:
Better using `$RANDOM` now that it have been enabled by default in OpenWrt instead of `$(random-numgen)`, do not forget to remove the dependency on that package too from the Makefile